src/_guides/language/evolution.md

```diff
@@ -180,131 +183,190 @@ turn into edit-time analysis errors.
 
 In Dart 2.12, **[Dart FFI][]** graduated from beta to the stable channel.
 
-### Dart 2.13
+### Dart 2.10
+_Released 1 October 2020_
+| [Dart 2.10 announcement](https://medium.com/dartlang/announcing-dart-2-10-350823952bd5)
 
-Dart 2.13 expanded support for **[type aliases][]** (`typedef`),
-which used to work only for function types
-but now work for any type.
-You can use the new name created with a type alias
-anywhere the original type could be used.
+Dart 2.10 added no new features to the Dart language.
+It did add an expanded [`dart` tool][dart-tool] similar to the
+Flutter SDK's [`flutter` tool][].
 
-Dart 2.13 also improved the struct support in **[Dart FFI][]**,
-adding support for inline arrays and packed structs.
+### Dart 2.9
```

Comment:
Since you added release dates for all the other entries, consider adding one here.

The [archive](https://dart.dev/get-dart/archive) specifies the binaries were released on Aug 5, 2020.

```suggestion
### Dart 2.9
_Released 5 Aug 2020_
```